src/store/modules/brushModule.js

```diff
@@ -13,35 +13,24 @@ const state = {
   imageBitmapCache: {}
 };
 
-const mutations = {
-  SET_DRAW_COLOR: (drawColorId) => {
-    state.draw = drawColorId;
-  },
-  SET_RADIUS: (radius) => {
-    state.radius = Math.min(
-      Math.max(radius, state.minRadius),
-      state.maxRadius
-    );
-  },
-  SET_MIN_RADIUS: (minRadius) => {
-    state.minRadius = minRadius;
-  },
-  SET_MAX_RADIUS: (maxRadius) => {
-    state.maxRadius = maxRadius;
-  },
-  SET_ALPHA: (alpha) => {
-    state.alpha = alpha;
-  },
-  SET_HIDDEN_BUT_ACTIVE_ALPHA: (alpha) => {
-    state.hiddenButActiveAlpha = alpha;
+const setters = {
+
+  /**
+   * Sets the brush radius, account for global min/max radius
+   *
+   * @param {*} radius
+   */
+  setRadius: (radius) => {
+    state.radius = Math.min(Math.max(radius, state.minRadius), state.maxRadius);
   },
 
   /**
+   * TODO: Should this be a init config property?
```

Comment:
Yes I think so. Hmm, what is a good way to account for 3rd party module configuration during the init call 🤔 ?

Reply:
Maybe not for `vNext`, but we can add a setup/config method as part of the "module interface" that gets called when we first create an instance of the store? 🤷‍♂️

Reply:
Sounds like a plan, following recent slack conversation, lets also leave this as is for now 👍 .